src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -29,10 +29,13 @@ def web_client_factory(cli_ctx, **_):
 
 
 def zip_contents_from_dir(dirPath, lang):
-    relroot = os.path.abspath(os.path.join(dirPath, os.pardir))
+    import tempfile
+    import uuid
+    relroot = os.path.abspath(tempfile.gettempdir())
```

Comment:
I believe you might have already tested this on both windows & Mac?

Reply:
Yeah just tested on windows and mac, haven't tested on any of the fancier os's

Reply:
Should I do a try catch, and default to the current behaviour?